### ISSUE TITLE
Segmentation fault on media import

### ISSUE THREAD

**Issue by stevemk14ebr** [Twanislas/docker-sonarr#1]
Frequent segmentation faults occur on 'import from existing media' events. This appears to happen during the disk scan

**Comment by Twanislas** [comment]
Hi !

What are you running this container on ?
Also, I remember libmediainfo causing segfaults on Alpine (which I'm not using anymore), you may want to check [this issue](https://github.com/Sonarr/Sonarr/issues/1462)

So far this is running since 1+ month on an X86_64 server without any issue.

**Comment by stevemk14ebr** [comment]
Hey sorry for late response, i was using docker for windows. I did run the alpine version, and tracked down your old posts mentioning you had that issue. So then i switched to your debian version and the issue persisted. I've since switched to this image: https://hub.docker.com/r/linuxserver/sonarr/ so i can't provide any more details, sorry. It's totally possible i misconfigured something, glad it's working for you and i appreciate your work on this!
